### DISCUSSION TITLE
如何在jest单元测试中初始化graph并添加node

### DISCUSSION THREAD

**Discussion by lpp1214** [antvis/X6#2749] [Q&A]
jest单元测试new Graph()初始化graph报错
![image](https://user-images.githubusercontent.com/19425553/194707581-ca4e4364-eac7-4ad2-86bc-d65a1fe4a9ec.png)
经排查elem.ts文件createElementNS方法调用doc.createElementNS(namespaceURI, tagName)处，生成对象为SVGElement没有getCTM方法（浏览器在此处生成的对象是SVGGraphicElement）

请问如何在jest单元测试中初始化graph并添加node

**Comment by tolerance-go** [comment]
same here
